azure_functions_worker/dispatcher.py

```diff
@@ -7,37 +7,35 @@
 
 import asyncio
 import concurrent.futures
+import importlib
+import inspect
 import logging
-import queue
-import threading
 import os
+import queue
 import sys
-import importlib
-import inspect
+import threading
+from asyncio import BaseEventLoop
+from logging import LogRecord
+from typing import Optional
 
 import grpc
 
 from . import bindings
+from . import constants
 from . import functions
 from . import loader
 from . import protos
-from . import constants
-
-from .constants import (
-    CONSOLE_LOG_PREFIX,
-    PYTHON_THREADPOOL_THREAD_COUNT,
-    PYTHON_THREADPOOL_THREAD_COUNT_DEFAULT,
-    PYTHON_THREADPOOL_THREAD_COUNT_MIN,
-    PYTHON_THREADPOOL_THREAD_COUNT_MAX
-)
-from .logging import error_logger, logger, is_system_log_category
-from .logging import enable_console_logging, disable_console_logging
+from .constants import (CONSOLE_LOG_PREFIX, PYTHON_THREADPOOL_THREAD_COUNT,
+                        PYTHON_THREADPOOL_THREAD_COUNT_DEFAULT,
+                        PYTHON_THREADPOOL_THREAD_COUNT_MAX,
+                        PYTHON_THREADPOOL_THREAD_COUNT_MIN)
+from .logging import disable_console_logging, enable_console_logging
+from .logging import error_logger, is_system_log_category, logger
 from .utils.common import get_app_setting
 from .utils.tracing import marshall_exception_trace
 from .utils.wrappers import disable_feature_by
-from asyncio import BaseEventLoop
-from logging import LogRecord
-from typing import Optional
+
+_TRUE = "true"
```

Comment:
Put it in constants.py would be a better place.

Reply:
Constants.py seem to be specific to more config-level or global setting and keeping constants within the modules they are used is a better approach.

If the constant is to be used in multiple modules, constants.py is better and we can refactor this when we see it being used in other places.

Reply:
Ah, I see, thanks for the explanation!